app/models/package.scala

```diff
@@ -122,7 +163,7 @@ package object models {
       (path.path, jsValue) match {
         case (Nil, _)                                                                  => JsError("path cannot be empty")
         case ((n: KeyPathNode) :: Nil, value: JsObject) if value.keys.contains(n.key)  => JsSuccess(value - n.key)
-        case ((n: KeyPathNode) :: Nil, value: JsObject) if !value.keys.contains(n.key) => JsError("cannot find value at path")
+        case ((n: KeyPathNode) :: Nil, value: JsObject) if !value.keys.contains(n.key) => JsSuccess(value)
```

Comment:
Is this copy and pasted? I presume this code comes pre-built with a new frontend. I wonder if this is a bug that was fixed? Might be worth reverting this change